src/extension/features/accounts/calculate-irr/index.js

```diff
@@ -0,0 +1,250 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteAccountsPage } from 'toolkit/extension/utils/ynab';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { getEntityManager } from 'toolkit/extension/utils/ynab';
+
+export class CalculateIRR extends Feature {
+  injectCSS() {
+    return require('./index.css');
+  }
+
+  shouldInvoke() {
+    let { selectedAccount } = controllerLookup('accounts');
+    return (
+      isCurrentRouteAccountsPage() &&
+      selectedAccount &&
+      !selectedAccount.onBudget &&
+      ynabToolKit.options.CalculateIRR !== 'disabled'
```

Comment:
you won't need this check once you rebase on master and change `default` to `false` in `settings.js`